src/traces/choropleth/plot.js

```diff
@@ -13,13 +13,16 @@ var Plotly = require('../../plotly');
 var d3 = require('d3');
 
 var Color = require('../../components/color');
+var Drawing = require('../../components/drawing');
+
+var getColorscale = require('../../components/colorscale/get_scale');
+var makeScaleFunction = require('../../components/colorscale/make_scale_function');
 var getTopojsonFeatures = require('../../lib/topojson_utils').getTopojsonFeatures;
 var locationToFeature = require('../../lib/geo_location_utils').locationToFeature;
 var arrayToCalcItem = require('../../lib/array_to_calc_item');
-var getColorscale = require('../../components/colorscale/get_scale');
-var makeScaleFunction = require('../../components/colorscale/make_scale_function');
```

Comment:
Mainly because I initially grouped all the components pieces together, then when I added `Drawing` next to `Color` I just left them :laughing: